packages/kernel/build.yaml

```diff
@@ -3,6 +3,16 @@ requires:
   category: "distro"
   version: ">=0"
 
+excludes:
+- ^/boot/grub2
```

Comment:
I think those two previously just slipped in, no real need to add them as part of the kernel package